### PR TITLE
Codesniffer updates to admin-views and views

### PR DESCRIPTION
**Pull request by borkweb**
Related:
* https://github.com/moderntribe/the-events-calendar/pull/119
* https://github.com/moderntribe/the-events-calendar/pull/120
* ~~https://github.com/moderntribe/the-events-calendar/pull/110~~

See: https://central.tri.be/issues/36908

### CODE REVIEW

File: src/admin-views/admin-update-message.php [hunk 1]
```diff
@@ -31,34 +31,34 @@
...
-				<a href="http://m.tri.be/nt" target="_blank"><?php _e( 'Pro' ); ?></a><br/>
-				<a href="http://m.tri.be/nu" target="_blank"><?php _e( 'Tickets' ); ?></a><br/>
-				<a href="http://m.tri.be/nx" target="_blank"><?php _e( 'Community Events' ); ?></a><br/>
-				<a href="http://m.tri.be/nv" target="_blank"><?php _e( 'Filters' ); ?></a><br/>
-				<a href="http://m.tri.be/nw" target="_blank"><?php _e( 'Facebook' ); ?></a><br/><br/>
+				<a href="http://m.tri.be/nt" target="_blank"><?php esc_html_e( 'Pro' ); ?></a><br/>
+				<a href="http://m.tri.be/nu" target="_blank"><?php esc_html_e( 'Tickets' ); ?></a><br/>
+				<a href="http://m.tri.be/nx" target="_blank"><?php esc_html_e( 'Community Events' ); ?></a><br/>
+				<a href="http://m.tri.be/nv" target="_blank"><?php esc_html_e( 'Filters' ); ?></a><br/>
+				<a href="http://m.tri.be/nw" target="_blank"><?php esc_html_e( 'Facebook' ); ?></a><br/><br/>
```
Comment (zbtirrell) [inline src/admin-views/admin-update-message.php]: These 5 translated strings don't have a domain.  That seems odd, but maybe it's fine?

File: src/admin-views/organizer-meta-box.php [hunk 2]
```diff
@@ -1,62 +1,62 @@
...
+	</td>
+	<td class="organizer-email">
+		<input tabindex="<?php tribe_events_tab_index(); ?>" type='text' id='OrganizerEmail' name='organizer[Email]' size='25' value='<?php echo isset( $_OrganizerEmail ) ? esc_attr( $_OrganizerEmail ) : ''; ?>' />
+	</td>
+</tr>
+<?php do_action( 'tribe_events_organizer_after_metabox', $post ); ?>
+
+<script type="text/javascript">
+	jQuery('[name=organizer\\[Organizer\\]]').blur(function () {
+		jQuery.post('<?php echo esc_url_raw( admin_url('admin-ajax.php') ); ?>',
```
Comment (zbtirrell) [inline src/admin-views/organizer-meta-box.php]: spacing

File: src/admin-views/admin-update-message.php (hunk 1 above)
Comment (borkweb) [inline src/admin-views/admin-update-message.php]: That is odd. When missing the domain, it defaults to the `default` domain. Though I find it unlikely that `Pro`, `Tickets`, etc are translated there.

**Comment by zbtirrell** [comment]
:+1:

**Pull request merged**